roles/tpa_single_node/tasks/importer/importer.yml

```diff
@@ -18,6 +18,15 @@
   delay: 5
   until: result.rc == 0
 
+- name: Set configmaps for importer
+  ansible.builtin.set_fact:
+    importer_configmaps: |
+      {{
+        [tpa_single_node_kube_manifest_dir + '/ConfigMaps/s3-trust-anchors.yaml']
+        if s3_trust_anchors_configmap_result is defined and s3_trust_anchors_configmap_result.path | length > 0
```

Comment:
What is the purpose of this if else block ?

Reply:
@desmax74, the s3-trust-anchors config map exists only when those root anchors are provided.